### DISCUSSION TITLE
[Feature request] Blacklist labels (ignoring dependabot/greenkeeper PRs) for Slack Integration

### DISCUSSION THREAD

**Discussion by vaibhavmule** [community/community#40994] [Actions]
Slack Integration

https://github.com/integrations/slack/issues/951

**Comment by ossanna16** [comment]
Hi there @vaibhavmule and welcome to our community! Can you please elaborate on your feature request a bit further? Thank you!

**Reply by FredericVaugeoisFlo** [reply]
The linked issue/PR explains why this is a needed feature. Right now, the teams and slack integration repos have been abandonned and we can't create any PR to address these issues since it's not open-source anymore. 

The feature requested here is simply to be able to filter out labels instead of only being able to filter-in labels (see [this section](https://github.com/integrations/slack#creating-a-filter)). I, personally, would like that to be done for Teams as well as Slack.
